beeflow/client/bee_client.py

```diff
@@ -53,6 +55,36 @@ def __init__(self, *args):
         self.args = args
 
 
+def warn(*pargs):
+    """Print a red warning message."""
+    typer.secho(' '.join(pargs), fg=typer.colors.RED, file=sys.stderr)
+
+
+def db_path():
+    """Return the client database path."""
+    bee_workdir = config_driver.BeeConfig.get('DEFAULT', 'bee_workdir')
+    return os.path.join(bee_workdir, 'client.db')
+
+
+def setup_hostname(start_hn):
+    """Set up front end name when beeflow core start is returned."""
+    db = bdb.connect_db(client_db, db_path())
+    db.info.set_hostname(start_hn)
+
+
+def check_hostname(curr_hn, stop=False):
+    """Check current front end name matches the one beeflow was started on."""
+    db = bdb.connect_db(client_db, db_path())
+    start_hn = db.info.get_hostname()
+    if start_hn != "" and curr_hn != start_hn:  # noqa: don't use set instead
+        warn(f'beeflow was started on "{start_hn}" and you are trying to '
```

Comment:
So currently the output is: 

```
Cannot connect to the beeflow daemon, is it running? Check the log at "/home/<username>/.beeflow/logs/beeflow.log".
beeflow was started on "hostname-fe3" and you are trying to run a command on "hostname-fe4".
```
This is great, but I think we should remove the old `Cannot connect to beeflow daemon` message.

Reply:
Got it!